components/file_upload/file_upload.jsx

```diff
@@ -354,14 +355,18 @@ export default class FileUpload extends PureComponent {
         const files = [];
         Array.from(droppedFiles).forEach((file, index) => {
             const item = items[index];
-            if (item && item.webkitGetAsEntry && item.webkitGetAsEntry().isDirectory) {
+            if (item && item.webkitGetAsEntry && (item.webkitGetAsEntry() === null || item.webkitGetAsEntry().isDirectory)) {
```

Comment:
Why `=== null` and not just a falsy check? Could `item.webkitGetAsEntry()` ever be `undefined`?

Reply:
Yes, it's not supported on edge nor safari, so it returns `null` and was then trying to access the property. It might be undefined in some other browser, but I haven't checked that.

Note that this is not part of the original ticket, but I was removing an error appearing on drag and drop, but not the one reported.